frontend/src/component/ui/ChangeLogModal.tsx

```diff
@@ -68,27 +83,28 @@ export class ChangeLogModal extends Component<Props, State> {
     );
   };
 
+  open = () => this.modalRef.current && this.modalRef.current.open();
+
+  close = () => this.modalRef.current && this.modalRef.current.close();
+
   render() {
-    if (!this.props.shouldShowModal) {
-      return <div />;
-    }
     return (
-      <div className={'modal-wrapper'}>
+      <Modal
+        ref={this.modalRef}
+        onClose={this.props.closeModal}
```

Comment:
```suggestion
        onClose={this.props.onClose}
```
We add `on` in front all events to make it easier to read.